exdb/tests.py

```diff
@@ -249,12 +249,58 @@ def create_sub_type(self):
     def create_org(self):
         return Organization.objects.create(name="Test Organization")
 
+    def create_keyword(self):
+        return Keyword.objects.create(name="Test Keyword")
+
+    def create_category(self):
+        return Category.objects.create(name="Test Category")
+
     def create_experience(self, exp_status):
         """Creates and returns an experience object with status of your choice"""
         return Experience.objects.create(author=self.test_user, name="E1", description="test description", start_datetime=self.test_date,
                                          end_datetime=(self.test_date + timedelta(days=1)), type=self.create_type(), sub_type=self.create_sub_type(), goal="Test Goal", audience="b",
                                          status=exp_status)
 
+    def create_experience_comment(self, exp):
+        """Creates experience comment, must pass an experience"""
+        return ExperienceComment.objects.create(
+            experience=exp, message="Test message", author=self.test_user, timestamp=self.test_date)
+
+
+class ModelCoverageTest(StandardTestCase):
+
+    def test_create_sub_type(self):
+        st = self.create_sub_type()
+        self.assertEqual(str(SubType.objects.get(pk=st.pk)), "Test Sub Type",
```

Comment:
I don't like that these are being tested with hard coded strings. This should be dry'ed out.